views/layout-form-edit.twig

```diff
@@ -65,6 +65,14 @@
                         {% set helpText %}{% trans "The Name of the Layout - (1 - 50 characters)" %}{% endset %}
                         {{ forms.input("name", title, layout.layout, helpText) }}
 
+                         {% set title %}{% trans "Start Time" %}{% endset %}
```

Comment:
I'm not sure what this does?

Reply:
That's clearly not supposed to be there, good catch! :P